### PR TITLE
Tri 301 refactore api

### PR DESCRIPTION
**Pull request by mkanal**
Changes made accoring to feedback in miro.

### CODE REVIEW

**Comment by ds-ext-kmassalski** [comment]
Change cancel method to PUT or PATCH (modifying resource)

**Comment by mkanal** [comment]
@ds-ext-kmassalski  cancel endpoint is now put.

File: api/irs-v0.2.yaml [hunk 1]
```diff
@@ -113,11 +113,10 @@ paths:
...
           schema:
             type: string
-            default: true
             maxLength: 16
             minLength: 1
-            pattern: ^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}$
-            example: 6c311d29-5753-46d4-b32c-19b918ea93b0 # Random Example
+            format: uuid
+            example: urn:uuid:6c311d29-5753-46d4-b32c-19b918ea93b0 # Random Example
         - description: "If true, the endpoint returns the uncompleted results of the parts tree."
```
Comment (ds-jkreutzfeld) [inline api/irs-v0.2.yaml:120]: PartsTree naming should no longer be used

**Review by ds-jkreutzfeld — COMMENTED** [review]
There are multiple uses of "PartsTree" in the document which should be replaced I think.

File: api/irs-v0.2.yaml (hunk 1 above)
Comment (mkanal) [reply api/irs-v0.2.yaml]: fixed

**Review by ds-jkreutzfeld — APPROVED** [review]

**Pull request merged**